DOKS-wordpress/README.md

```diff
@@ -150,6 +152,255 @@ Finally, you need to setup the trusted sources between your MySQL database and y
 
 Please visit [How to Secure MySQL Managed Database Clusters](https://docs.digitalocean.com/products/databases/mysql/how-to/secure/) for more details.
 
+## Introduction to the OpenEBS Dynamic NFS Provisioner
+
+When we deploy a stateful application in a Kubernetes cluster, we create and access DigitalOcean Block Storage Volumes by creating a PersistentVolumeClaim as part of our deploymment using the default StorageClass named do-block-storage.
+DigitalOcean Block Storage Volumes are mounted as read-write by a single node (RWO). Additional nodes cannot mount the same volume. The data content of a PersistentVolume can not be accessed by multiple Pods simultaneously.
+
+Horizontal pod autoscaling (HPA) policies are used to scale the WordPress Pods in a dynamically StatefulSet. As a result, Kubernetes schedules different Pods with the same data content as the other replicas. In this specific case, not only is the same content served by WordPress replicas, any update of the content, such as new images, user uploaded files, or plugins, need to be synchronously made available to the other replicas.
+
+This is why WordPress requires a volume that can be mounted as read-write by many nodes (RWX), in our case, WordPress replica Pods.
+
+NFS (Network File System) is a commonly used solution to provide RWX volumes on block storage. This server offers a PersistentVolumeClaim (PVC) in RWX mode so that multiple web applications can access the data in a shared fashion. In many cases, cloud block storage providers or OpenEBS volumes are used as persistent backend storage for these NFS servers to provide a scalable and manageable RWX shared storage solution.
+
+OpenEBS Dynamic NFS Provisioner allows users to create an NFS PV that sets up a new Kernel NFS instance for each PV on top of the user's choice of backend storage.
+
+**Note:**
+
+Please visit [OpenEBS](https://openebs.io/) for more details.
+
+### Installing and configuring the OpenEBS Dynamic NFS Provisioner
+
+In this section, you will install the OpenEBS Dynamic NFS Provisioner on your Kubernetes cluster using the [OpenEBS Helm Chart](https://github.com/openebs/dynamic-nfs-provisioner). For our purposes we will be installing and configuring only the dynamic nfs provisioner since that's what we need.
+
+First, add the `Helm` repository:
+
+```console
+helm repo add openebs-nfs https://openebs.github.io/dynamic-nfs-provisioner
+
+helm repo update
+```
+
+Next, create a YAML file `openEBS-nfs-provisioner.yaml` to override the default helm values:
+
+```yaml
```

Comment:
Please avoid embedding the full values file in the README. It's big and hard to follow. 

Then, at a later time and if it's relevant for the guide, you can provide important sections from the values file and highlight the important settings.